### DISCUSSION TITLE
Sending Domain and From address is invalid though we have verified DKIM from ESP

### DISCUSSION THREAD

**Discussion by nagireddybi** [postalserver/postal#2158] [Help with using Postal]
Hello Team
We are using postal as one of our major SMTP servers, We add postal SMTP to different plans. users under the plan verify their domains and from email addresses to send emails from Email Sending Software (It is like mailwizz or acelle). But when users sending emails it is throwing 

Sender domain or from address invalid 
If we use the domain verified in postal it works
But domains verified in email sending software, it doesn't work but PowerMTA works 
Error is below
"Expected response code 250 but got code "530", with message "530 From/Sender name is not valid "

Do we need to do any extra configuration at postal, to make postal allow DKIM verified domains to send emails through postal

Thanks

**Comment by willpower232** [comment]
The credential you're using needs to be in the same "server" as the domains that have been added and verified.
